pages/core_ucb/core_ucb_authorities/core_ucb_person_page.rb

```diff
@@ -0,0 +1,6 @@
+class CoreUCBPersonPage < CorePersonPage
```

Comment:
@Seanhaoxiang - You shouldn't need to create one of these empty classes anymore.